crowd-server/crowd_server/apps/image_label/models.py

```diff
@@ -1,3 +1,41 @@
 from django.db import models
+from django.contrib.auth.models import User
```

Comment:
Try to use `from django.contrib.auth import get_user_model`. This way if you change the `User` and used a custom `User`, your code will not crash.